release-gap-package

```diff
@@ -452,7 +452,11 @@ rm -rf .git* .hg* .cvs* .circleci
 rm -f .mailmap
 rm -f .appveyor.yml .codecov.yml .travis.yml azure-pipelines.yml
 # remove TeX leftovers
-find -E doc -regex '.*\.(aux|bbl|blg|brf|idx|ilg|ind|log|out|pnr|toc|tst)' -exec rm -f {} +
+if [ "$(uname)" == "Darwin" ]; then
+  find -E doc -regex '.*\.(aux|bbl|blg|brf|idx|ilg|ind|log|out|pnr|toc|tst)' -exec rm -f {} +
```

Comment:
Probably should do `find doc -name '*.aux' -o -name '*.bbl' -o ...`

Reply:
`find doc -name '*.aux' -o -name '*.bbl' -o -name '*.blg' -o -name '*.brf' -o -name '*.idx' -o -name '*.ilg' -o -name '*.ind' -o -name '*.log' -o -name '*.out' -o -name '*.pnr' -o -name '*.toc' -o -name '*.tst'` will do